### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by wale-prog**
In this pull request, I have implemented the following:
- Created a method called `current_user` in the ApplicationController that returns the first user from the database
- Created form to create posts on behalf of the `current_user`
- Create an input field for comments on behalf of the `current_user`
- Allow users to add likes to posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
```
Comment (rloterh) [inline app/controllers/comments_controller.rb:1]: - [ ] `new` action is missing in your comments controller. Kindly note that this module requires that you explicitly declare both `new` and `create` actions for comments controller. Kindly ensure to improve upon the same, which should be similar to:
```ruby
# controllers/comments_controller.rb

class CommentsController < ApplicationController
  ...

  def new
    @comment = Comment.new
  end

  def create
  
    ...
  end


  private 
  def comment_params
    params.require(:comment).permit(:text)
  end

	...
end

File: config/routes.rb [hunk 2]
```diff
@@ -3,6 +3,9 @@
 
   # Defines the root path route ("/")
   resources :users, only: [:show, :index] do 
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:create]
```
Comment (rloterh) [inline config/routes.rb]: - [ ] As per the comments controller actions `new` and `create`, kindly ensure that there is a pair of corresponding routes in the `config/routes.rb` file for creating comments, which should yield:
```ruby
   POST   /posts/:post_id/comments(.:format)     comments#create
 GET   /posts/:post_id/comments/new(.:format)     comments#new
```

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Wale, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Form working correctly


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
- I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by sole contributor. 
- It's recommended that you add an MIT license, or any other similar license to your ReadMe.
Implementing the above change(s) will make your ReadMe appear more professional.

<br>
Cheers and Happy coding!👏 💻 🍷
<br><br>
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb [hunk 3]
```diff
@@ -3,6 +3,9 @@
 
   # Defines the root path route ("/")
   resources :users, only: [:show, :index] do 
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:create, :new]
+      resources :likes, only: [:create]
```
Comment (codepantha) [inline config/routes.rb:8]: - [ ] An error occurs when trying to reach the new comments endpoint which shows the new comment form. This is a requirement just like the previous reviewer stated. Kindly implement it 👍 
```ruby
GET   /posts/:post_id/comments/new(.:format)     comments#new
```
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @wale-prog 👋, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors

## Suggested changes
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by BenMukebo — APPROVED** [review]
### Hi @wale-prog, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**